examples/tutorials/validate_transformers_turorial.ipynb

```diff
@@ -0,0 +1,2310 @@
+{
```

Comment:
Do we need to save `self.prev_interactions` for `ValidationMetrics` callback? I don't see why we need it

Reply:
Removed from `self` variable.